frontend/app/routes/$lang+/_public+/apply+/$id+/personal-information.tsx

```diff
@@ -76,7 +78,11 @@ export async function loader({ context: { session }, params, request }: LoaderFu
     id: state.id,
     csrfToken,
     meta,
-    defaultState: state.personalInformation,
+    defaultState: {
+      ...state.personalInformation,
+      email: state.communicationPreferences?.email ?? state.personalInformation?.email,
+      confirmEmail: state.communicationPreferences?.confirmEmail ?? state.personalInformation?.confirmEmail,
```

Comment:
HERE: Should only fallback to `communicationPreferences?.email ` if `personalInformation?.email` is not defined.

```suggestion
      email: state.personalInformation?.email ?? state.communicationPreferences?.email,
      confirmEmail: state.personalInformation?.confirmEmail ?? state.communicationPreferences?.confirmEmail,
```